src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -233,6 +234,7 @@ class BootstrapUsers(_AnsiblePlaybookAlias):
     def run(self, args, unknown_args):
         environment = get_environment(args.env_name)
         args.playbook = 'deploy_stack.yml'
+        args.factory_auth = True
         public_vars = environment.public_vars
         root_user = public_vars.get('commcare_cloud_root_user', 'root')
         unknown_args += ('--tags=users', '-u', root_user)
```

Comment:
This stuff (adding `'-u', public_vars.get('commcare_cloud_root_user', 'root')` to the ansible args) should happen on account of `--use-factory-auth`, so this should get added in the same place that the rest of that functionality happens.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1702/commits/c875390478ba7efb6492f5af4023ae5a50babc22

https://github.com/dimagi/commcare-cloud/pull/1702/commits/898af473bfe427544070dea1c554873d8842bce5

https://github.com/dimagi/commcare-cloud/pull/1702/commits/27542ff516ac6c7c35a38eca57d88a62818e4d8f

(This part is still buggy, I need to keep debugging)